MekHQ/src/mekhq/gui/dialog/RetirementDefectionDialog.java

```diff
@@ -173,8 +184,8 @@ private void initComponents(boolean doRetirement) {
             }
             if (null == contract) {
                 instructions += "\n\nDays since last retirement roll: "
-                        + Utilities.countDaysBetween(rdTracker.getLastRetirementRoll().getTime(),
-                                hqView.getCampaign().getDate());
+                        + Math.toIntExact(ChronoUnit.DAYS.between(rdTracker.getLastRetirementRoll(),
```

Comment:
Why `toInt` here and above, Is `long` not going to work?

Reply:
Force of habit to move from long to int, swapped them both to long because they do work as intended as longs.